pkg/volume/util/atomic_writer.go

```diff
@@ -252,7 +251,7 @@ func validatePath(targetPath string) error {
 	if targetPath == "" {
 		return fmt.Errorf("invalid path: must not be empty: %q", targetPath)
 	}
-	if path.IsAbs(targetPath) {
+	if filepath.IsAbs(targetPath) {
```

Comment:
filepath.IsAbs() only returns true for paths with disk prefix (e.g. `C:\\path`). For linux absolute paths (e.g. `/a/b`), it will return false on Windows.

So it's better to still use `path.IsAbs()` here.

Reply:
On Windows, does `path.IsAbs()` work correctly?

Reply:
Hmm, sorry. Just making another verification. Both of them are not working as expected:

- path.IsAbs("/a/b") return true
- but path.IsAbs("\\a\\b") return false (this path is actually the result of `filepath.Clean("/a/b")`)

while

- filepath.IsAbs return false for both "/a/b" and "\\a\\b"

So we should probably use `filepath.IsAbs()` and add another check of path prefixes "/" and "\".

Reply:
Use `strings.HasPrefix()` instead?